src/components/Form.js

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import { React, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { AddBook } from '../redux/books/books';
 
 function Form() {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const id = uuidv4();
+  const dispatch = useDispatch();
+
+  const addBook = () => {
+    const data = {
+      title, author, id,
+    };
+    console.log(data);
```

Comment:
- [ ] Please kindly remove this `console.log` statement, `console. log()` will reduce your program's performance since it takes computational time. ℹ️